### ISSUE TITLE
Eq(9) in paper

### ISSUE THREAD

**Issue by KiAkize** [Howal/DNL-Object-Detection#2]
1. How does the first equal sign in Equation 9 hold?
2. Why can $\lambda_i$ be omitted in subsequent discussions?

**Comment by houhouhouhou11** [comment]
hello, can you run this project successfullly? thx

**Comment by KiAkize** [comment]
> hello, can you run this project successfullly? thx

Sorry i don't understand mmcv framework:( Why could not provide pure pytorch code?

**Comment by houhouhouhou11** [comment]
i think mmdetectio is a  framework that wrap mask_rcnn and fater rcnn ..

**Comment by Howal** [comment]
1. This equation is initially described at Page.5, and we put the proof at Appendix.C.
2. λ is a normalization scaler to keep the sum over Ω equals 1.

**Comment by KiAkize** [comment]
Obviously, lambda_i is a complex expression including q_i, k_j and so on, which can not be omitted directly.  

    
  




On Tue, Sep 29, 2020 at 4:17 PM +0800, "Zhuliang Yao" <notifications@github.com> wrote:
















This equation is initially described at Page.5, and we put the proof at Appendix.C.

λ is a normalization scaler to keep the sum over Ω equals 1.

**Comment by ljwztc** [comment]
> 1. This equation is initially described at Page.5, and we put the proof at Appendix.C.
> 2. λ is a normalization scaler to keep the sum over Ω equals 1.

Thanks for your impressive work. But I'm still confused about the λ. How does it calculate? @Howal
